### PR TITLE
doc: Add PR for TSC scope

### PR DESCRIPTION
**Pull request by mhdawson**
Initial version of the document to define the scope
of the TSC.

Content was started in https://github.com/nodejs/TSC/issues/113
and after discussion in the recent TSC meeting
it was agreed that now a PR would make sense so people
can comment and refine the language.

### CODE REVIEW

File: doc/TSCScope.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Subject to the contraints defined in the foundation
+charter, these are the areas that the TSC has autonomy
+in making decisions (some of which can be delegated to
+other groups, for example the CTC).  Outside of these
+areas the TSC will collaborate with the board
+when making decisions:
+
+* Code/Doc/ changes to any of the repositories under github/nodejs
```
Comment (nebrius) [inline doc/TSCScope.md]: Is this meant to say/imply `Code/Doc/etc`? Are there certain types of changes that wouldn't be allowed?

File: doc/TSCScope.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* Standards and processes covering contributions to repositories under
```
Comment (nebrius) [inline doc/TSCScope.md]: Is the intention that working groups would be covered by this clause? Some of them certainly make commits on the repo (docs, diagonistics, etc) some are more process oriented (inclusivity WG), but does this cover everything? I'm thinking of cases like the evangelism WG.

File: doc/TSCScope.md (hunk 1 above)
Comment (mhdawson) [inline doc/TSCScope.md]: yes intended to cover all changes to the repos.

File: doc/TSCScope.md (hunk 2 above)
Comment (mhdawson) [inline doc/TSCScope.md]: Yes.  The TSC delegates some of its autonomy to the Working Groups but it is through the TSC that this autonomy originates.

File: doc/TSCScope.md (hunk 1 above)
Comment (nebrius) [inline doc/TSCScope.md]: in that case, can we change it to read `Changes to the contents of any of the...` or something similar? This way it's clear that we have scope over everything in the repos, not just certain types of content.

File: doc/TSCScope.md (hunk 2 above)
Comment (nebrius) [inline doc/TSCScope.md]: What do you think about something along the lines of `Standards and processes covering contributions to and interactions with repositories and related Internet properties under github/nodejs`? I feel that `contributions to repositories` is rather narrow and may not cover certain types of interactions, e.g. IRC, evangalism WG's interactions on non-GitHub mediums (Twitter, etc). I think these fall under our scope.

File: doc/TSCScope.md (hunk 1 above)
Comment (mhdawson) [inline doc/TSCScope.md]: Sounds good to me, will do an update sometime mid next week to address all comments up to that point in time.

File: doc/TSCScope.md (hunk 2 above)
Comment (mhdawson) [inline doc/TSCScope.md]: Seems reasonable to be.  Sounds good to me, will do an update sometime mid next week to address all comments up to that point in time.

**Comment by mhdawson** [comment]
Pushed additional commit to address comments so far.

File: doc/TSCScope.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  under github/nodejs.
+* Standards and processes covering contributions and interactions
+  with repositories and related Internet properties under
+  github/nodejs.
+* Infrastructure and usage of build tools for components
+  built from github/nodejs.  
+* Release types/schedules/frequency/delivery vehicles
+* Which external components Node will depend on and how they
+  will be bundled into Node distributions. (Examples include
+  node-gyp, v8, opensll, etc.). This does not include bringing
```
Comment (addaleax) [inline doc/TSCScope.md]: typo: openssl

**Comment by mhdawson** [comment]
I think we can just close this.  Doing that reopen if thats not the case.

**Pull request closed without merging**